src/nginxconfig/generators/index.js

```diff
@@ -40,6 +40,7 @@ import magentoConf from './conf/magento.conf';
 import joomlaConf from './conf/joomla.conf';
 import dockerComposeYaml from './yaml/dockerCompose.yaml';
 import dockerConf from './ext/docker';
+import shareQuery from '../util/share_query.js';
```

Comment:
```suggestion
import shareQuery from '../util/share_query';
```
Just a tiny nitpick here, we don't need the extension for the file